client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -353,7 +355,7 @@ class JetpackThankYouCard extends Component {
 		if ( reasons && reasons.length > 0 ) {
 			reason = translate( 'We can\'t modify files on your site.' );
 			this.trackConfigFinished( 'calypso_plans_autoconfig_error_filemod', { error: reason } );
-		} else if ( ! selectedSite.hasMinimumJetpackVersion ) {
+		} else if ( selectedSite.hasMinimumJetpackVersion === false ) {
```

Comment:
Hmm, I don't see how this change can help here... as `hasMiniumJetpackVersion` is computed, if there's no jetpack version number synced yet, its value is going to be false anyway

Reply:
The value would be `null` if the don't have the number version yet. 

https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors.js#L940

Reply:
ooooooh, cr*p!! I was looking at the old computed version: 

https://github.com/Automattic/wp-calypso/blob/master/client/lib/site/jetpack.js#L25

good enough! :D